styles.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+    font-family: "Figtree-ExtraBold";
+    src: url('./assets/fonts/static/Figtree-ExtraBold.ttf') format('truetype');
```

Comment:
For some reason these imported fonts are blurry in my Safari browser (left), but look just fine in Chrome (right). Don't know off the top of my head what to do 🤷 Just pointing it out

![Screenshot 2025-04-12 at 7 12 47 PM](https://github.com/user-attachments/assets/3f97b051-997a-4729-b3c6-2394bf562623)